tools/classifier/requirements.txt

```diff
@@ -1,4 +1,4 @@
 # Add your dependencies here
 
 # Required for all unstract tools
-unstract-sdk~=0.53.1
+unstract-sdk~=0.54.0rc2
```

Comment:
@muhammad-ali-e shouldn't this be rc5? That's the latest

Reply:
@muhammad-ali-e Let us make it rc5 everywhere to be consistent.

Reply:
it will take the latest rc due to `~=`

Reply:
updated to rc5